pkg/gadgets/trace/dns/types/dns.go

```diff
@@ -42,6 +42,12 @@ type Event struct {
 	Uid uint32 `json:"uid" column:"uid,template:uid,hide"`
 	Gid uint32 `json:"gid" column:"gid,template:gid,hide"`
 
+	SrcIP    string `json:"srcIP,omitempty" column:"srcIP,template:ipaddr"`
+	DstIP    string `json:"dstIP,omitempty" column:"dstIP,template:ipaddr"`
+	SrcPort  uint16 `json:"srcPort,omitempty" column:"srcPort,template:ipport"`
+	DstPort  uint16 `json:"dstPort,omitempty" column:"dstPort,template:ipport"`
+	Protocol string `json:"protocol,omitempty" column:"proto,maxWidth:5"`
```

Comment:
IMO, they should be hidden by default. It's too much info to be displayed by default.

Reply:
Made these 5 columns `visible=false`.